src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Identity/CommandHandler/RightToBeForgottenCommandHandlerTest.php

```diff
@@ -178,6 +175,11 @@ public function an_identity_may_not_be_forgotten_twice(): void
             ->with(new IsEqual($nameId), new IsEqual($institution))
             ->andReturn($this->createIdentity($identityId->getIdentityId()));
 
+        $this->sensitiveDataService
+            ->shouldReceive('forgetSensitiveData')
```

Comment:
I would expect some test to fail now because you actually clear the name / email.

Do you think that should/could be tested?

Reply:
I've looked into this but projections are not tested, I presume this was a design choice because you should be able to replay a projection. I've checked if I could add some tests while at it, but it is hard to do now because of how the compiler passes are implemented for replays (when using an acceptance test) and the repositories are completely mocked away in the current test setup which would take me a long time to setup which I think wouldn't justify the time that would take at the moment.